occlum/run_occlum_passed_tests.sh

```diff
@@ -1,60 +1,65 @@
 #!/bin/bash
 
-set -x
-
+#set -x
 TEST_BIN_DIR=/opt/occlum/gvisor_syscall_tests
-BLOCKLIST_DIR=blocklist/
-TEST_LIST=occlum_test_list.txt
+BLOCKLIST_DIR=../blocklist
+TEST_LIST=../occlum_test_list.txt
 TESTS=0
 PASSED_TESTS=0
 RED='\033[0;31m'
 NC='\033[0m'
 RESULT=0
 FILTER=""
 
-get_blocklisted_subtests() {
+get_blocklist_subtests(){
 	if [ -f $BLOCKLIST_DIR/$1 ]; then
-        FILTER=$(cat $BLOCKLIST_DIR/$1 | tr '\n' ':')
-        return 0
+		FILTER=$(cat $BLOCKLIST_DIR/$1 | tr '\n' ':')
+		return 0
 	else
-        FILTER=""
+		FILTER=""
 		return 1
-    fi
+	fi
 }
 
-run_one_test() {
-    ret=""
-
-    if [ -f $TEST_BIN_DIR/$1 ]; then
-		get_blocklisted_subtests $1
-
-        occlum new occlum_workspace
-        pushd occlum_workspace
-        cp $TEST_BIN_DIR/$1 image/bin
-
-        occlum build
-        occlum run /bin/$1 --gtest_filter=-$FILTER
-        ret=$?
-        popd
-        rm -rf occlum_workspace
-    else
-        echo "Warning: test does not exist"
-        ret=1
-    fi
-
-    return $ret
+run_one_test(){
+	ret=""
+	if [ -f $TEST_BIN_DIR/$1 ]; then
+		get_blocklist_subtests $1
+		occlum exec /bin/$1 --gtest_filter=-$FILTER
+		ret=$?
+	else
+		echo "Warning: test does not exit"
+		ret=1
+	fi
+	return $ret
 }
 
+occlum new occlum_workspace
```

Comment:
What if there exists the occlum_workspace? Maybe delete the old occlum_workspace before creating new workspace.